UI/lib/ui-header.html

```diff
@@ -39,23 +39,15 @@
         window.alert("[% text('Warning:  Your password will expire in [_1]', pw_expires)%]");
     </script>
     [% END %]
+    <script src="js-src/lsmb/webpack.loaderConfig.js"></script>
```

Comment:
How do other projects solve this problem?

Reply:
Removed